Arm/Memory/SeparateAutomation.lean

```diff
@@ -403,6 +473,35 @@ def MemSeparateProp.ofExpr? (e : Expr) : Option MemSeparateProp :=
     .some { sa, sb }
   | _ => none
 
+/-- info: Prod.mk.{u, v} {α : Type u} {β : Type v} (fst : α) (snd : β) : α × β -/
+#guard_msgs in #check Prod.mk
+
+/-- info: List.cons.{u} {α : Type u} (head : α) (tail : List α) : List α -/
+#guard_msgs in #check List.cons
+
+/-- info: List.nil.{u} {α : Type u} : List α -/
+#guard_msgs in #check List.nil
```

Comment:
```suggestion
```
We don't need these at this point, I think. It's fairly clear what's going on at `MemPairwiseSeparateProof.ofExpr?`.

Reply:
@shigoel I prefer to leave them in, since it at least helps me sanity check what's going on --- code is read a lot more than it's written!